routes/helpers/render.js

```diff
@@ -61,7 +61,15 @@ let render = {
     vars.qualifiedURL = config.qualifiedURL;
 
     vars.urlPath = url.parse(req.originalUrl).pathname;
-
+    //return to path, shouldn't change on sigin or register pages
+    if (typeof (req.query.returnTo) == 'undefined'){
+        vars.returnTo = '/';
+        }    
+    else {
+        vars.returnTo = req.query.returnTo;
+    }
+    console.log('view', view);
+    console.log('returnTo',vars.returnTo);
```

Comment:
These shouldn't be in the PR (we can add debug statements for stuff we really want to log, but this should not be necessary to log).

Reply:
Noted.  These are personal debug statements I meant to delete.